dask/dataframe/_compat.py

```diff
@@ -11,7 +11,8 @@
 PANDAS_GT_133 = PANDAS_VERSION >= Version("1.3.3")
 PANDAS_GT_140 = PANDAS_VERSION >= Version("1.4.0")
 PANDAS_GT_150 = PANDAS_VERSION >= Version("1.5.0")
-PANDAS_GT_200 = PANDAS_VERSION.major >= 2  # Also true for nightly builds
+PANDAS_GT_200 = PANDAS_VERSION.major >= 2
+PANDAS_GT_210 = PANDAS_VERSION.release >= (2, 1, 0)
```

Comment:
@phofl I saw https://github.com/pandas-dev/pandas/pull/52018 is in the 2.1 milestone, so _I think_ the changes in this PR are correct. Just wanted to double check that this deprecation won't show up in a future `pandas=2.0.x` release

Reply:
Yes, this will come in 2.1. generally we don’t backport deprecations if we can avoid it.

there will be a couple more axis=1 deprecations soonish (just as a heads up)